operator/internal/controller/auto/workspace_controller.go

```diff
@@ -340,6 +338,9 @@ func labelsForStatefulSet(w *autov1alpha1.Workspace) map[string]string {
 }
 
 func newStatefulSet(ctx context.Context, w *autov1alpha1.Workspace, source *sourceSpec) (*appsv1.StatefulSet, error) {
+	// TODO: get from configuration
+	workspaceAgentImage := "pulumi/pulumi-kubernetes-operator-v2:" + version.Version
```

Comment:
Maybe this should be read from an environment variable.

Reply:
I'll try to hook this up to the downward API.